src/vaadin-combo-box-mixin.html

```diff
@@ -580,16 +580,32 @@
      */
     _onEscape(e) {
       if (this.autoOpenDisabled) {
```

Comment:
I think the implementation here has several issues:

- Escape is propagated when auto open is disabled. Don't understand why this branching exists. As long as the dropdown is open, then pressing Escape should only close the dropdown, but not propagate the event further.
- When the clear button is visible, then clearing the text field value with Escape also causes the event to be propagated. Clearing the text field value should not cause dialogs to close.

Reply:
All of those cases are resolved now.